components/Footer/Nav.js

```diff
@@ -1,29 +1,50 @@
 import Image from "next/image";
 import Link from "next/link";
 import React from "react";
+import { useState } from "react";
 import { StyledNav } from "./StyledNav";
 
 export default function Nav() {
+  const [isActive, setIsActive] = useState(0);
+
   return (
     <StyledNav>
       <Link href="/">
-        <Image
-          className="home"
-          src="/appstore-outlined.svg"
-          width={36}
-          height={36}
-          alt="home icon"
-        />
+        {isActive === 0 ? (
```

Comment:
you may use true / false instead of 0 / 1

Reply:
In later US' i going to extend the navbar therefore i will use it like that.